adagrams/game.py

```diff
@@ -1,11 +1,123 @@
+import random
+
+# Wave 1
+
+
 def draw_letters():
-    pass
+    """
+    Uses dict to create a list of the letter pool.
+    Shuffles list and returns slice of first 10 items. 
+    """
+    LETTER_POOL = {
+        'A': 9,
+        'B': 2,
+        'C': 2,
+        'D': 4,
+        'E': 12,
+        'F': 2,
+        'G': 3,
+        'H': 2,
+        'I': 9,
+        'J': 1,
+        'K': 1,
+        'L': 4,
+        'M': 2,
+        'N': 6,
+        'O': 8,
+        'P': 2,
+        'Q': 1,
+        'R': 6,
+        'S': 4,
+        'T': 6,
+        'U': 4,
+        'V': 2,
+        'W': 2,
+        'X': 1,
+        'Y': 2,
+        'Z': 1
+    }
+
+    letter_list = []
+    for letter in LETTER_POOL:
+        counter = 0
+        while counter < LETTER_POOL[letter]:
+            letter_list.append(letter)
+            counter += 1
+
+    random.shuffle(letter_list)
+
+    return letter_list[0:10]
+
+# Wave 2
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    """
+    Makes copy of letter bank list.
+    Checks to see if all letters in the words are in the letter bank.
+    """
+
+    letter_bank_copy = []
+    for item in letter_bank:
+        letter_bank_copy.append(item)
```

Comment:
We can eliminate the for loop and reduce these 3 lines into 1 by using the `copy` method to generate a copied list from `letter_bank`.
```suggestion
letter_bank_copy = letter_bank.copy()
```